tests/exampleTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\CloudFunctions\Tests;
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Tests for when this package is installed in a vendored directory
+ *
+ * @group gcf-framework
+ */
+class exampleTest extends TestCase
+{
+    private static $containerId;
+    private static $imageId;
+
+    public static function setUpBeforeClass(): void
+    {
+        if ('true' === getenv('TRAVIS')) {
+            self::markTestSkipped('These tests do not pass on travis');
+        }
+
+        $exampleDir = __DIR__ . '/../examples/hello';
+        self::$imageId = 'test-image-' . time();
+
+        $cmd = 'composer update -d ' . $exampleDir;
```

Comment:
Why `composer update` here?

Reply:
just in case locally you update the `composer.json` and forget to remove your `composer.lock`